### PR TITLE
Add zgen instructions

### PR DESCRIPTION
**Pull request by unixorn**
* Add zgen instructions
* Make spacing between sections consistent
* Minor phrasing changes in README.md

### CODE REVIEW

**Comment by MichaelAquilina** [comment]
Thanks for this :)

**Comment by unixorn** [comment]
I also added it to the [awesome-zsh-plugins](https://github.com/unixorn/awesome-zsh-plugins) list.

File: README.md [hunk 1]
```diff
@@ -22,20 +22,32 @@ On Ubuntu simply install from the standard repositories:
...
 antigen bundle MichaelAquilina/zsh-autoswitch-virtualenv
 ```
-if you want to set a default virtual environment then you can also export `AUTOSWITCH_DEFAULTENV` in
+
+Installing with Zgen
+--------------------
+
+Installing with [Zgen](https://github.com/tarjoilija/zgen) is also easy! Add the following line to your `.zshrc`:
+
+```zgen load unixorn/tumult.plugin.zsh```
```
Comment (MichaelAquilina) [inline README.md]: Just noticed this. This seems like its the incorrect path. Wouldn't
`zgen load MichaelAquilina/zsh-auto-switch-virtualenv` be correct?

Comment (unixorn) [inline README.md]: Yes, that is me having a brain fart when I copied the instructions from another plugin. Sorry about that.

Comment (MichaelAquilina) [inline README.md]: its ok, I should have been the one to catch it :)

**Pull request merged**